api/core/types.proto

```diff
@@ -19,6 +19,8 @@ enum FluxObjectKind {
     KindHelmRelease    = 5;
     KindCluster        = 6;
     KindOCIRepository  = 7;
+    KindProvider       = 8;
+    KindAlert          = 9;
```

Comment:
For my personal notes, do you remember why you needed to add this?

Reply:
Great question. I had to go back to look - the `kind` getter in the flux object class uses FluxObjectKind - so everything will crash when you try to use Provider and Alert as Objects unless they're in here - for whatever it's worth it also crashes the YamlView which I think is the only place where the Provider and Alert kinds will actually be looked at

` get kind(): FluxObjectKind | undefined {
    if (this.obj.kind) {
      return FluxObjectKind[addKind(this.obj.kind)];
    }
  }`

Reply:
Ah. Hmpf. Of course. Thank you!